doctrine-mapping.xsd

```diff
@@ -412,9 +419,16 @@
     <xs:sequence>
       <xs:any minOccurs="0" maxOccurs="unbounded" namespace="##other"/>
     </xs:sequence>
-    <xs:attribute name="class" type="xs:string" use="required" />
+    <xs:attribute name="class" type="orm:CLASSPATH" use="required" />
   </xs:complexType>
 
+  <xs:simpleType name="CLASSPATH" id="CLASSPATH">
+    <xs:restriction base="xs:token">
+      <xs:pattern value="([a-zA-Z_$][a-zA-Z\d_$\\]*\.)*[a-zA-Z_$][a-zA-Z\d_$\\]*" id="CLASSPATH.pattern">
```

Comment:
The regex for class names is something like `/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]+$/` - not just ascii